crypto/src/test_utils/epoch_observer.rs

```diff
@@ -40,3 +41,9 @@ impl EpochObserver for TestEpochObserver {
         guard.observed_epochs.push((conversation_id, epoch));
     }
 }
+
+impl Debug for TestEpochObserver {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        f.write_str("TestEpochObserver")
+    }
+}
```

Comment:
We can probably use `derive_more` crate, which we're already using elsewhere, to derive such simple impls, like

```
#[derive(derive_more::Debug)]
#[debug("TestEpochObserver")]
struct TestEpochObserver(....)
```